frontend/src/components/public_group/AmountPicker.js

```diff
@@ -44,7 +44,7 @@ export default class AmountPicker extends React.Component {
         )}
 
         <div className='max-width-1 mx-auto' style={{position: 'absolute', left: '0', right: '0', margin: '-25px auto'}}>
-          {!hasStripe && (
+          {(!hasStripe || (hasPaypal && hasStripe)) && (
```

Comment:
In the current setup, both `hasStripe` and `hasPaypal` should never be true at the same time.

`hasPaypal` requires a different authentication than what a host gives us normally to pay out expenses. This is because Paypal has different APIs (and authentication) for receiving payments and paying out money. When we setup Paypal as a receiving mechanism, we store it in `ConnectedAccounts` table. When we setup Paypal to payout expenses, we store it in the `PaymentMethods` table.

It's going to be confusing for our users because they'll end up auth'ing paypal twice. I hope there's a way to do this in one go but I haven't had a chance to look at it yet.

@arnaudbenard please correct if I'm mistaken about any of this.